itou/employee_record/management/commands/process_asp_report_file.py

```diff
@@ -0,0 +1,98 @@
+import json
+import logging
+import os.path as os_path
+
+from django.core.management.base import BaseCommand
+from rest_framework.renderers import JSONRenderer
+
+from itou.employee_record.models import EmployeeRecord, EmployeeRecordBatch
+from itou.employee_record.serializers import EmployeeRecordSerializer
+
+
+class Command(BaseCommand):
+    """
+    Manually process an employee record ASP report file
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        handler = logging.StreamHandler(self.stdout)
+
+        self.logger = logging.getLogger(__name__)
+        self.logger.propagate = False
+        self.logger.addHandler(handler)
+
+        self.logger.setLevel(logging.INFO)
+
+    def add_arguments(self, parser):
+        """
+        Command line arguments
+        """
+        parser.add_argument("--file", dest="input_file", type=open, required=True, help="ASP input file")
+
+    def handle(self, *args, **options):
+        """
+        Fixes employee record "in-between" state in case of the crash of CRON jobs
```

Comment:
```suggestion
        Fixes employee record "in-between" state in case of the crash of CRON jobs.
        The employee records already sent to the ASP but with an error code are inserted in our database again
```